Documentation/DomainModel/Entities.md

```diff
@@ -0,0 +1,209 @@
+# Domain Entities
+
+## Identity Context
+
+### Administrator
+Table name: `phplist_admin`
+
+An **administrator** can log in to the system and is allowed to administer
+selected **lists** (as the owner), send **campaigns** to these lists and edit
+**subscribers**.
+
+Administrators are not subscribers. If administrators would like to subscribe
+to lists, they need to have a separate subscriber account.
+
+### AdministratorAttribute
+Table name: `phplist_adminattribute` or `phplist_admin_attribute` 
+
+What does this do? How does this work?
+
+### AdministratorPasswordRequest
+Table name: `phplist_admin_password_request`
+
+What does this do? How does this work?
+
+### AdministratorToken
+Table name: `phplist_admintoken`
+
+This table contains the API tokens for **administrators**. Those API tokens are
+used for access to the REST API.
+
```

Comment:
I guess they can be used for the API. At the moment, these are used for XSRF protection.